### DISCUSSION TITLE
Don't format certain section

### DISCUSSION THREAD

**Discussion by lars-vc** [nvim-neorg/neorg#1304] [Q&A] [answered]
I want to write something like the following
```
    start -> dummy -> dummy 1
                   -> dummy 2
```
in norg but when I format the dummy 2 goes to the beginning of the line, as the spec defines it to.
However, is there anyway to avoid this behaviour? I thought a code block could fix this, but alas.
If there isn't a way of doing this, I would suggest code blocks with no language to not be formatted or to allow for a 
```
@code no-format
    start -> dummy -> dummy 1
                   -> dummy 2
@end 
```
approach.

**Comment by pysan3** [comment, marked as answer]
With the current state of the plugin, it is not possible. Don't run `gg=G` on that file is the only solution. 

Some guys are working on a new treesitter parser and when that's done, there'll be a _real_ formatter [norg-fmt](https://github.com/nvim-neorg/norg-fmt) so you'd want to wait for updates on that module.

**Reply by lars-vc** [reply]
Oh that's unfortunate.
